images/master/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM google/debian:wheezy
```

Comment:
cAdvisor and Heapster we base off of [busybox](https://github.com/google/cadvisor/blob/master/deploy/Dockerfile) and use the host [volume mount](https://github.com/GoogleCloudPlatform/heapster/blob/master/deploy/heapster-controller.yaml) of `/etc/ssl/certs` for the SSL certs